indico/modules/events/abstracts/fields.py

```diff
@@ -30,13 +29,28 @@
 from indico.modules.events.abstracts.util import serialize_abstract_person_link
 from indico.modules.events.contributions.models.persons import AuthorType
 from indico.modules.events.fields import PersonLinkListFieldBase
+from indico.modules.users.models.users import User
 from indico.util.decorators import classproperty
 from indico.util.i18n import _
 from indico.web.forms.base import AjaxFieldMixin
 from indico.web.forms.fields import MultipleItemsField, JSONField
 from indico.web.forms.widgets import JinjaWidget, SelectizeWidget
 
 
+def serialize_user(user):
+    return {
+        'id': user.id,
+        'name': user.name
+    }
+
+
+def get_users_in_roles(data):
+    user_ids = {user_id for user_roles in data.viewvalues()
```

Comment:
I'd wrap before `for` to have each loop on its own line. At least I usually try to wrap everything as soon as it doesn't fit in a single line and IMO it's more readable. Especially with the ridiculous order Python uses for multiple-for comprehensions I think it's not a good idea to have a somewhat unrelated loop and the `user_id` in the same line.